File: docs/components_page/components/layout/no_gutters.jl
using DashBootstrapComponents, DashHtmlComponents

row = dbc_row(
    [
        dbc_col(html_div("One of three columns")),
        dbc_col(html_div("One of three columns")),
        dbc_col(html_div("One of three columns")),
    ],
    className = "g-0",
);

File: docs/components_page/components/layout/no_gutters.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

row <- dbcRow(
  list(
    dbcCol(htmlDiv("One of three columns")),
    dbcCol(htmlDiv("One of three columns")),
    dbcCol(htmlDiv("One of three columns"))
  ),
  className = "g-0"
)
